detekt-rules-libraries/build.gradle.kts

```
@@ -2,8 +2,23 @@ plugins {
     id("module")
 }
 
+val generatedLibrariesConfig: Configuration by configurations.creating {
+    isCanBeConsumed = false
+    isCanBeResolved = true
+}
+
 dependencies {
     compileOnly(projects.detektApi)
     testImplementation(projects.detektTest)
     testImplementation(libs.assertj)
+
+    generatedLibrariesConfig(
+        project(projects.detektGenerator.path, "generatedLibrariesConfig")
+    )
+}
+
+tasks.named<ProcessResources>("processResources").configure {
```

oh, it seems we don't need this, Task.inputs is visible anyway.
```suggestion
tasks.named("processResources").configure {
```

---

indeed 👍